web/package.json

```diff
@@ -4,6 +4,7 @@
 	"scripts": {
 		"dev": "svelte-kit dev --host 0.0.0.0 --port 3002",
 		"build": "svelte-kit build",
+		"start": "node ./build/index.js",
```

Comment:
If you are using the make file from the root of the project to run `make dev`, it will spin up everything in development mode for you. So you don't need to start separate services/

Reply:
Oh I haven't seen that, thank you for the hint!

Reply:
Can we remove this since it is not needed?

Reply:
Actually I'm using it to run the backend in the IDE without running the docker stack :)
If you prefer I can remove it tho